src/Atomizer/Renderer.php

```diff
@@ -401,6 +402,10 @@ private function columnOptions(Serializer $serializer, AbstractColumn $column):
             $options['precision'] = $column->getPrecision();
         }
 
+        if ($options['default'] === $column::DATETIME_NOW || ($options['default'] instanceof Fragment && (string)$options['default'] === $column::DATETIME_NOW)) {
```

Comment:
```suggestion
        $default = $options['default'];
        if ($column::DATETIME_NOW === ($default instanceof \Stringable ? (string)$default : $default)) {
```

I think this fragment is little prettier :)